### PR TITLE
update overview of how PX works with DCOS framework

### PR DESCRIPTION
**Pull request by ferrantim**

### CODE REVIEW

File: scheduler/mesosphere-dcos/cassandra.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 redirect_from: "/dcos-cassandra.html"
 ---
 
 * TOC
 {:toc}
 
-This guide will help you to install the [Cassandra](https://portworx.com/use-case/cassandra-docker-container/) service on your DCOS cluster backed by PX volumes for persistent storage.
+DC/OS provides a Cassandra service that makes it easy to deploy and manage Cassandra on Mesosphere DC/OS. This guide will help you to install and run the [containerized Cassandra](https://portworx.com/use-case/cassandra-docker-container/) service backed by Portworx volumes for [persistent DCOS storage](https://portworx.com/use-case/persistent-storage-dcos/).  With [Portworx backing your Cassandra cluster](https://docs.portworx.com/applications/cassandra.html), you can 
+
+* Recovery faster during a failure
```
Comment (disrani-px) [inline scheduler/mesosphere-dcos/cassandra.md]: Recover faster

File: scheduler/mesosphere-dcos/cassandra.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+* Achieve higher density by running multiple Cassandra rings on the same DC/OS hosts and
+* Simply deployments
```
Comment (disrani-px) [inline scheduler/mesosphere-dcos/cassandra.md]: Simplify

File: scheduler/mesosphere-dcos/cassandra.md [hunk 3]
```diff
@@ -85,25 +89,19 @@ marathon                     10.0.4.21                   True     1    1.0   102
...
 ```
 
-## Hyper-convergence and Failover
-When each Cassandra task is first launched, they create the required PX volumes. These volumes are created with data local 
-to the node where they are first launched.
- 
-On subsequent launches of the same pod for example in the case of a failover, the framework queries Portworx to figure out
-where the data for the volume resides and uses this to decide where the pod should be launched.
+## Hyperconvergence
+Running your Cassandra task on the same host as its data provides the best performance.  This is called hyperconvergence and it is supported by the DC/OS Cassandra when using Portworx. When each Cassandra task is first launched, they create the required PX volumes. These volumes are created with data local to the node where they are first launched.
```
Comment (disrani-px) [inline scheduler/mesosphere-dcos/cassandra.md]: supported by the DC/OS Cassdanra framework when using Portworx

**Comment by ferrantim** [comment]
Thanks for the review @disrani-px I addressed all the comments and merged.

**Pull request merged**